code/script.js

```diff
@@ -1,13 +1,85 @@
 // All the DOM selectors stored as short variables
-const chat = document.getElementById('chat')
+const chat = document.getElementById("chat")
+const nameForm = document.getElementById("name-form")
+const inputwrapper = document.getElementById("input-wrapper")
 
 // Global variables, if you need any, declared here
 
 // Functions declared here
 
+const nameInput = document.getElementById("name-input")
+
+const handleNameInput = (event) => {
+  event.preventDefault()
+
+  showMessage(nameInput.value, "user")
+
+  nameInput.value = ""
+
+  setTimeout(()=> question1(), 1000)
+
+  
+
+  }
+
+  const question1 = () => {
+    showMessage("Hur ofta vill du spara?", "bot")
+    inputwrapper.innerHTML = `
+    <button id="button1">Lite varje månad</button>
+    <button id="button2">Ett engångsbelopp</button>`
+    document.getElementById("button1")
+    .addEventListener("click", () => question2())
+    document.getElementById("button2")
+    .addEventListener("click", () => answer1())
+
+  }
+
+  const question2 = () => {
```

Comment:
Maybe this function could be named `askForAmount` or something? You can keep track of which functions takes care of which question by writing comments above:
```
// Question 2
const askForAmount = () => {
```

Reply:
With that said, naming things are one of the hardest things there is in programming 🙈 Think of functions as actions, so the name should contain at least one verb.